lms/templates/ccx/coach_dashboard.html

```diff
@@ -17,6 +17,16 @@
 
 <%include file="/courseware/course_navigation.html" args="active_page='ccx_coach'" />
 
+## Translators: ccx_connector is url for CCX Connector application for managing creation of CCXs.
+<%
+   use_ccx_con_error = _("You cannot create a CCX based on this course because the Advanced Setting "
```

Comment:
We should move this variable assignment into `views.py` and pass it here in `context`